tests/EventListener/AffiliateTokenListenerTest.php

```diff
@@ -0,0 +1,37 @@
+<?php
+
+namespace App\EventListener;
```

Comment:
Is it ok to keep tests in the same namespace as whole application? In Symfony docs I see `App\Tests\...`.

Reply:
Fixed the namespace